tests/KdybyTests/CsobPaymentGateway/HttpClientMock.php

```diff
@@ -71,9 +71,15 @@ private function resolveTargetFile($method, $url, $headers, $body)
 	{
 		$parsedUrl = parse_url($url);
 		$path = isset($parsedUrl['path']) ? $parsedUrl['path'] : '';
-
-		list(,,, $resource, $action) = explode('/', $path);
-		$endpoint = $resource . '/' . $action;
+		if(substr_count($path, '/') === 3)
```

Comment:
ok, so I actually came up with something: 6b89a07 :) can you rebase this onto master please?